src/engines/fx/chorus/chorus.cpp

```diff
@@ -5,42 +5,41 @@
 #include "util/iterator.hpp"
 #include "util/utility.hpp"
 
-#include "chorus.faust.hpp"
-
 namespace otto::engines {
 
-  using namespace ui;
-  using namespace ui::vg;
+    using namespace ui;
+    using namespace ui::vg;
 
-  struct ChorusScreen : EngineScreen<Chorus> {
-    void draw(Canvas& ctx) override;
-    bool keypress(Key key) override;
-    void rotary(RotaryEvent e) override;
+    struct ChorusScreen : EngineScreen<Chorus> {
+        void draw(Canvas& ctx) override;
+        bool keypress(Key key) override;
+        void rotary(RotaryEvent e) override;
 
-    using EngineScreen<Chorus>::EngineScreen;
-  };
+        using EngineScreen<Chorus>::EngineScreen;
+    };
 
-  Chorus::Chorus()
-    : EffectEngine(name, props, std::make_unique<ChorusScreen>(this)),
-      faust_(std::make_unique<faust_chorus>(), props)
-  {}
+    Chorus::Chorus()
+            : EffectEngine("Chorus", props, std::make_unique<ChorusScreen>(this))
+    {
+      // Initialize LPF for param smoothening
+      lpf.type(gam::SMOOTHING);
+      lpf.freq(1000);
 
+      props.delay.on_change().connect([this](float delay) {
+        chorus.delay(delay);
+      });
 
-  audio::ProcessData<2> Chorus::process(audio::ProcessData<1> data)
-  {
-    return faust_.process(data);
-  }
+      props.rate.on_change().connect([this](float rate) {
+        chorus.freq(rate);
+      });
 
-  // SCREEN //
+      props.feedback.on_change().connect([this](float fbk) {
+        chorus.fbk(fbk);
+      });
 
-  void ChorusScreen::rotary(ui::RotaryEvent ev)
-  {
-    auto& props = engine.props;
-    switch (ev.rotary) {
-    case Rotary::blue: props.delay.step(ev.clicks); break;
-    case Rotary::green: props.rate.step(ev.clicks); break;
-    case Rotary::yellow: props.deviation.step(ev.clicks); break;
-    case Rotary::red: props.depth.step(ev.clicks); break;
+      props.depth.on_change().connect([this](float depth) {
+        chorus.depth(depth);
+      });
     }
```

Comment:
Thats the one - delete this line and it should work

Reply:
Done. Waiting for travis.